projects/nico778/proposal-template.md

```diff
@@ -0,0 +1,43 @@
+# Proposal: Delta Mush
+
+nico778
+
+01.01.2022
+
+## Abstract
+
+Delta Mush smoothes a deformed mesh while not losing the detail of the original mesh.
+This project tries to implement the algorithm presented in the paper by Mancewicz et al.
+
+## Motivation
+
+Character rigs that are used in animations should represent a realistic and natural model of the character. To achieve this, the deformations, or even short sequences of an information, should avoid looking unrealistic and unnatural to the human eye.
+Since the clean up of these bad deformations relies on a lot of manual labour. 
+Delta Mush solves this by maintaining the detail of the original mesh in the deformed mesh.
+Delta Mush caught my eye during the lecture, because I was not aware that there is already a solution for this problem.
+I'm very interested to see how the algorithm works in detail. 
+This work could be interesting to anybody involved in the creation of animations(in games/films). There might also be application potential in the field of engineering, specifically various CAD tasks.
+
+## Proposal
+
+The goal is to implement the Delta Mush algorithm that is described in the paper by Mancewicz et al..
+This project is geometry processing related because it solves a practical problem by modifying the mesh structure
+There are a few implementation on github, mostly in C++ and with Maya integration.
+I have not seen any js implementations using three.js.  
+
+## Implementation
+
+Since I am already used to typescript and three.js, these will be used in this project.
+
+Milestones:
+1. the smoothing of the original undeformed mesh
+2. creating the local rest coordinate system
+3. creating the local current coordinate system
```

Comment:
Interesting. I was wondering where the deformation information comes from? Are you planning to use a armature?

Reply:
@JohannesMerkt I tried to do the deformation with Skeleton/Bones in threejs, but that turned into very messy code and did not work properly. For now I am using an armature with 5 bones in blender. The deformed mesh doesn't quite look like the example in the slides though. Maybe this can be improved with constraints for the bones. I am planning to export the deformed tube as  an .obj file and work with that.

@MarlQ That is true indeed, hopefully there is still a little bit of time available for that.
Good that you mention that, I have to take a closer look at that.